### PR TITLE
User can record time through play-stop icons in navbar

### PR DESCRIPTION
**Pull request by LauraRe**
#### Description
_As a User,
In order to record my time live
I would like to be able to use the play and stop buttons_

Changes proposed in this pull request:

* record time through play and stop icons in navbar
* auto-fill input fields with recorded times when pressing on stop icon
#### What I have learned working on this feature: 
* how to use cypress clock and tick methods
* how to use componentDidUpdate in React

#### Screenshots:
image

### CODE REVIEW

File: cypress/integration/userCanSaveTimeNavbar.feature.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+				"X-AUTH-TOKEN": "api_kitten"
+			}
+		})
+			.get('button').contains('Get started here!').click()
+			.get(':nth-child(1) > .ui > input').type('susan_super')
+			.get(':nth-child(2) > .ui > input').type('api_kitten')
+			.get('button').contains('Login').click()
+	})
+
+	it('when user visits the page and logs in', () => {
```
Comment (vick3d) [inline cypress/integration/userCanSaveTimeNavbar.feature.js]: I don't think we need to do this test in this feature as well

File: cypress/integration/userCanSaveTimeNavbar.feature.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+		const begin =  moment.tz("2019-03-28 12:00", "Europe/Stockholm").valueOf()
+		const stub = cy.stub()
+
+		cy.on('window:alert', stub)
+		cy.clock(begin)
+		cy.get('#play').click()
+		cy.then(() => {
+			expect(stub.getCall(0)).to.be.calledWith("Started recording");
+		})
+		// 1000 = 1 second
```
Comment (vick3d) [inline cypress/integration/userCanSaveTimeNavbar.feature.js]: remove comment

Comment (LauraRe) [reply cypress/integration/userCanSaveTimeNavbar.feature.js]: the comment was for future users..but we can remove it

File: cypress/integration/userCanSaveTimeNavbar.feature.js (hunk 1 above)
Comment (perhegg) [reply cypress/integration/userCanSaveTimeNavbar.feature.js]: done

Comment (LauraRe) [reply cypress/integration/userCanSaveTimeNavbar.feature.js]: done

File: cypress/integration/userCanSaveTimeNavbar.feature.js (hunk 2 above)
Comment (LauraRe) [reply cypress/integration/userCanSaveTimeNavbar.feature.js]: done

Comment (perhegg) [reply cypress/integration/userCanSaveTimeNavbar.feature.js]: done

**Pull request merged**